package.json

```diff
@@ -39,16 +35,23 @@
     "greenkeeper[bot] <greenkeeper[bot]@users.noreply.github.com>"
   ],
   "scripts": {
-    "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader test/*.js && nyc report",
+    "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader build/test/*.js && nyc report",
     "docs": "jsdoc -c .jsdoc.js",
     "generate-scaffolding": "repo-tools generate all",
     "lint": "eslint src/ system-test/ test/ conformance/",
     "prettier": "prettier --write src/*.js src/**/*.js test/*.js test/**/*.js system-test/*.js system-test/**/*.js conformance/*.js conformance/**/*.js",
     "publish-module": "node ../../scripts/publish.js firestore",
-    "system-test": "mocha system-test/*.js --timeout 600000",
-    "conformance": "mocha conformance/*.js --no-timeouts",
-    "test-no-cover": "mocha test/*.js --no-timeouts",
-    "test": "npm run cover"
+    "system-test": "mocha build/system-test/*.js --timeout 600000",
+    "conformance": "mocha build/conformance/*.js --no-timeouts",
+    "test-no-cover": "mocha build/test/*.js --no-timeouts",
```

Comment:
`--no-timeouts` can cause real big misses in the tests - doesn't need to happen here, but we should file a follow up to get rid of it.